### PR TITLE
cabal.project: Fix/improve running of tests

### PR DESCRIPTION
**Pull request by erikd**

### CODE REVIEW

**Comment by erikd** [comment]
Now I just have to make Nix accept it.

**Comment by erikd** [comment]
bors merge

File: cabal.project [hunk 1]
```diff
@@ -18,96 +18,51 @@ package cardano-config
...
-package cardano-binary
+package ouroboros-consensus
   tests: False
 
-package cardano-sl-x509
+package ouroboros-consensus-cardano
   tests: False
 
-package cardano-crypto-class
+package ouroboros-consensus-cardano
```
Comment (mrBliss) [inline cabal.project:49]: Duplicate

File: cabal.project [hunk 2]
```diff
@@ -18,96 +18,51 @@ package cardano-config
...
-  tests: False
+package cardano-cli
+  tests: True
 
-package byron-spec-chain
-  tests: False
+package cardano-node
+  tests: True
 
+-- The following is needed because Nix is doing something crazy.
```
Comment (mrBliss) [inline cabal.project:39]: A pity that we need this.

**Pull request merged**